### PR TITLE
Fixed changes in Note

### PR DESCRIPTION
**Pull request by muskan2622**
### Functionality:
Fixed delete functionality and loader in Quick Note

### Solution:
Delete button is working properly.

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
1. Go ton Worktez
2. Go to QuickNotes, make notes and check it.

### CODE REVIEW

File: src/app/body/quick-notes/quick-notes.component.ts [hunk 1]
```diff
@@ -72,6 +72,13 @@ export class QuickNotesComponent implements OnInit {
   }
 
   deleteNote(docId: string) {
+    for(let i=0; i<this.quickNotes.notes.length; i++){
+      console.log(this.quickNotes.notes[i].DocId)
```
Comment (Pendyala-thanvi) [inline src/app/body/quick-notes/quick-notes.component.ts:76]: Remove this console.

File: src/app/body/quick-notes/quick-notes.component.ts [hunk 2]
```diff
@@ -72,6 +72,13 @@ export class QuickNotesComponent implements OnInit {
...
+      
+      if(this.quickNotes.notes[i].DocId==docId){ 
+      delete this.quickNotes.notes[i];
```
Comment (Pendyala-thanvi) [inline src/app/body/quick-notes/quick-notes.component.ts:79]: We don't need to completely delete the note, we're just marking it as deleted for now. So we don't need this.

File: src/app/body/quick-notes/quick-notes.component.ts (hunk 1 above)
Comment (muskan2622) [reply src/app/body/quick-notes/quick-notes.component.ts]: Removed

**Comment by viv9k** [comment]
Change base branch

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**